runtime/altair/src/weights/pallet_preimage.rs

```diff
@@ -2,23 +2,23 @@
 //! Autogenerated weights for `pallet_preimage`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 32.0.0
-//! DATE: 2024-06-24, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2024-06-28, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
 //! HOSTNAME: `runner`, CPU: `AMD EPYC 7763 64-Core Processor`
-//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("altair-local")`, DB CACHE: 1024
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("development")`, DB CACHE: 1024
 
 // Executed Command:
 // target/release/centrifuge-chain
 // benchmark
 // pallet
-// --chain=altair-local
+// --chain=altair
 // --steps=50
 // --repeat=20
 // --pallet=pallet_preimage
 // --extrinsic=*
 // --wasm-execution=compiled
 // --heap-pages=4096
-// --output=/tmp/runtime/altair/src/weights/pallet_preimage.rs
```

Comment:
Are we adding `development` weights into `altair`?

Why does the output path change to `development` if the file is under Altair? 🤔

Reply:
Eagle eyes! Altair benchmarks didnt work until last night so I just used the development ones because they are more or less the same and I just updated gov related benchmarks. However, I will do another benchmark execution and push true Altair weights.

Reply:
Makes sense! Actually, I guess we could just just compute weights only for `centrifuge` and use them for all runtimes

Reply:
Yes I agree. In the end, we want to ensure Centrifuge weights are feasible.

So are you fine with merging the PR without recomputing the weights for Altair as well here?